app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,22 @@
+class OrdersController < ApplicationController
+
+  def index
+    @orders = current_user.orders
+  end
+
+  def create
+    order = current_user.orders.create(total_price: @cart.total)
+    submit_order(order)
+    session.delete(:cart)
+    flash[:success] = "Mischief is coming your way, you have successfully placed your order!"
+    redirect_to orders_path
+  end
+
+  def submit_order(order)
```

Comment:
I could potentially see this as a PORO, but I'm wondering if it would be better for now as an instance method on Order?

If the functionality evolves such that you are doing things like sending a confirmation, I might see a stronger case for a PORO, but right now it seems that all of the functionality is strongly related to the order and its relationship to its items.